code/modules/antagonists/traitor/uplink_handler.dm

```diff
@@ -6,6 +6,8 @@
 /datum/uplink_handler
 	/// The owner of this uplink handler.
 	var/datum/mind/owner
+	/// The antag datum that we are linked to.
+	var/datum/antagonist/traitor/traitor_datum
```

Comment:
sure, it's fine for easier access

Reply:
It's just important to remember that they aren't necessarily a traitor.

Reply:
continuing on from this point, I would prefer for this to be a list of permanent objectives set by the traitor datum rather than a link to the traitor datum, for modularity purposes.

Reply:
Changed to using a list on the uplink handler that points to the same objective list the traitor datum uses.